### ISSUE TITLE
Upgrade for PHP 7.4 container

### ISSUE THREAD

**Issue by jennybeaumont** [humanmade/altis-local-server#243]

**Comment by roborourke** [comment]
We still need to tag an updated container which is waiting on the PHP 7.4 upgrade PR to be merged. This is basically done though and can happen soon.

**Comment by nathanielks** [comment]
[PHP 7.4 has now been merged into master!](https://github.com/humanmade/docker-wordpress-php/pull/30) I have also created the `v4.0-branch` which will build images for use with our infrastructure. Docker hub can use whatever.

**Comment by joehoyle** [comment]
@nathanielks should we also tag a v4-release, https://hub.docker.com/r/humanmade/altis-local-server-php/tags?page=1&ordering=last_updated I think will automatically build new tags, so even if we build a `4.0.0-alpha1` we should get a docker image for it too to switch `local-server` over to.

**Comment by roborourke** [comment]
I tagged `4.0.0-alpha` as a pre-release of the PHP container, we can try this out for Local Server once that and the dev container version are built.

Would be good to get the XDebug upgrade done as part of this too #242

**Comment by joehoyle** [comment]
👍 FYI I've been using php 7.4 for a while with that container (had my own tagged version) with local-server and working well for me.

**Comment by nathanielks** [comment]
Nice, thanks @roborourke ! As noted in #242, Xdebug 3.0.1 is in the latest versions of our PHP containers. I believe this issue can now be closed?

**Comment by roborourke** [comment]
Not quite, still need to make the PR to update the docker-compose.yml, will do that now once I've tested it locally, think I'll need to do some modifications for xdebug.

**Comment by nathanielks** [comment]
Brilliant, I'll reassign this to you 👍
